components/buttons.css

```
@@ -0,0 +1,79 @@
+.buttons {
```

This name is confusing as this is a section and not a button. How about:
```suggestion
.energy-rating {
```
or something like that?